example/pubspec.lock

```diff
@@ -407,16 +407,16 @@ packages:
     dependency: "direct overridden"
     description:
       path: "../packages/reactive_ble_mobile"
-      relative: true
+      relative: false
     source: path
-    version: "5.0.2"
+    version: "5.0.3"
   reactive_ble_platform_interface:
-    dependency: transitive
+    dependency: "direct overridden"
     description:
-      name: reactive_ble_platform_interface
-      url: "https://pub.dartlang.org"
-    source: hosted
-    version: "5.0.2"
+      path: "../packages/reactive_ble_platform_interface"
+      relative: true
```

Comment:
I, personally, prefer not to commit dependency overrides (or straight path-dependencies). Unless you have an explicit desire to change what's in the repo from how it was to this, I'd suggest to not commit any overrides and not use straight path-dependencies.